src/index.js

```diff
@@ -82,6 +82,17 @@ export default {
     DEBUG = flag;
   },
 
+  /**
+   *
+   *
+   * @param {string} region
+   * Accepted values: ROOFTOP, RANGE_INTERPOLATED, GEOMETRIC_CENTER, APPROXIMATE
+   */
+  setLocationType(locationType) {
+    LOCATIONTYPE = locationType;
```

Comment:
`LOCATIONTYPE` is not defined in the scope.

Reply:
Pushed